### PR TITLE
Add SIG operator team to governance doc

### PR DESCRIPTION
**Pull request by periklis**
**What this PR does / why we need it**:
This PR appends the SIG operator team to the official goverance doc as discussed here:
https://docs.google.com/document/d/1dSyDwUHpFdAjdS46X4RNx9OkewY2JimRWq3Xs39LScU/edit#

### CODE REVIEW

**Review by cyriltovena — APPROVED** [review]
LGTM on my side waiting for @RichiH to merge

File: docs/sources/community/governance.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ This document describes the rules and governance of the project. It is meant to
 
 - **The Loki project**: The sum of all activities performed under this governance, concerning one or more repositories or the community.
 
+- **The Loki Operator sub-project**: The sum of all activities performed under this governance, concerning the repository subfolder `operator` or the community.
```
Comment (RichiH) [inline docs/sources/community/governance.md:18]:
```suggestion
- **The Loki SIG (Special Interest Group) Operator**: The sum of all activities performed under this governance, concerning the repository subfolder `operator` or the community.
```

File: docs/sources/community/governance.md [hunk 2]
```diff
@@ -59,6 +61,13 @@ The current team members are:
 - Tom Braack - [sh0rez](https://github.com/sh0rez) ([Grafana Labs](https://grafana.com/))
 - Tom Wilkie - [tomwilkie](https://github.com/tomwilkie) ([Grafana Labs](https://grafana.com/))
 
+The current SIG operator team members are:
```
Comment (RichiH) [inline docs/sources/community/governance.md:64]:
```suggestion
The current Loki SIG Operator team members are:
```

**Review by RichiH — CHANGES_REQUESTED** [review]
LGTM including suggested cleanup.

**Comment by KMiller-Grafana** [comment]
@RichiH Can you please re-review https://github.com/grafana/loki/pull/4977?

**Review by RichiH — APPROVED** [review]
LGTM

**Pull request merged**